src/Components/HeaderCategory.jsx

```diff
@@ -1,7 +1,21 @@
 import React, { Component } from 'react';
-import { Menu } from 'semantic-ui-react'
+import { Menu, Divider, Container } from 'semantic-ui-react'
 
-const categories = ['News', 'Arts', 'Books', 'Business', 'Food', 'Opinion', 'Politics', 'Real Estate', 'Science', 'Sports', 'Style', 'Tech', 'Travel']
+const categories = [
```

Comment:
Just a little comment. Categories are 13 without the `news`.

https://glocal-news.herokuapp.com/api/v1/categories